pkg/image/definition.go

```diff
@@ -48,6 +49,28 @@ type Suma struct {
 	GetSSL        bool   `yaml:"getSSL"`
 }
 
+type Hauler struct {
+	ContainerImages []ContainerImage `yaml:"images"`
+	HelmCharts      []HelmChart      `yaml:"charts"`
+	Files           []File           `yaml:"files"`
+}
+
+type ContainerImage struct {
+	Name string `yaml:"name"`
+	Key  string `yaml:"key"`
+}
+
+type HelmChart struct {
+	Name    string `yaml:"name"`
+	RepoURL string `yaml:"repoURL"`
+	Version string `yaml:"version"`
+}
+
+type File struct {
+	Name string `yaml:"name"`
+	Path string `yaml:"path"`
```

Comment:
Can this be a local path or is it always an HTTP URL?

Reply:
resolving as a result of the removal of hauler file handling